File: app/src/main/java/com/bq/zowi/controllers/KitonNetworkController.java
package com.bq.zowi.controllers;

import android.support.annotation.Nullable;
import com.bq.zowi.models.networkModels.KitonIsAliveResponseNetworkModel;
import rx.Single;

/* JADX INFO: loaded from: classes.dex */
public interface KitonNetworkController {
    Single<KitonIsAliveResponseNetworkModel> sendIsAliveToKiton(String str, @Nullable String str2);
}

File: zowi-core/src/main/kotlin/com/bq/zowi/api/KitonNetworkController.kt
package com.bq.zowi.api

import com.bq.zowi.models.networkModels.KitonIsAliveResponseNetworkModel
import io.reactivex.Single

interface KitonNetworkController {
    fun sendIsAliveToKiton(macAddress: String, zowiAppId: String?): Single<KitonIsAliveResponseNetworkModel>
}
